docs/developer/checkout/performance.mdx

```diff
@@ -80,11 +80,19 @@ For lazy operations, only requesting certain fields will trigger prices recalcul
 
 It takes at least one field to trigger recalculation.
 
-These fields are (for Checkout & Order in Draft or Unconfirmed status)
+These fields are:
+
+**For Checkout**
+- `totalPrice`, `subtotalPrice`
+- `shippingPrice`
+- lines `totalPrice`
+- lines `unitPrice`
+
+**For Order in Draft or Unconfirmed status**
 - `total`, `subtotal` & `undiscountedTotal`
 - `undiscountedShippingPrice`, `shippingPrice` & `shippingTaxRate`
 - `discounts`
-- lines `total` & `undiscountedTotalPrice`
+- lines `totalPrice` & `undiscountedTotalPrice`
```

Comment:
Order object has `total` https://docs.saleor.io/api-reference/orders/objects/order#ordertotaltaxedmoney---

Reply:
But it's a `OrderLine` which has `totalPrice`

Reply:
You're right, I have "sałatka jarzynowa" brain